corehq/apps/data_interfaces/views.py

```diff
@@ -265,6 +267,10 @@ class CaseGroupCaseManagementView(DataInterfaceSection, CRUDPaginatedViewMixin):
     deleted_items_header = ugettext_noop("Removed Cases:")
     new_items_header = ugettext_noop("Added Cases:")
 
+    @method_decorator(require_permission(Permissions.edit_messaging))
+    def dispatch(self, request, *args, **kwargs):
+        return super(CaseGroupCaseManagementView, self).dispatch(request, *args, **kwargs)
```

Comment:
Why does case group need edit messaging permissions?

(I don't know what CaseGroup is used for, but I assume its something to do with cases and its not explicit why someone needs messaging permission)

Do we need to do the same for `CaseGroupListView`?

Reply:
This came up in QA-- ticket [here](https://dimagi-dev.atlassian.net/browse/QA-3575) and Case Groups is found under the messaging tab which is why it's restricted here. The edit_messaging check for `CaseGroupListView` is found elsewhere so that is properly checked, but for `CaseGroupCaseManagementView` the user could have still entered in the exact URL before this change. 
![image](https://user-images.githubusercontent.com/36681924/135109405-e23f1c66-711d-4783-b3fd-3474a62369bd.png)

Reply:
Cool, thanks.